docs/metrics/extending-the-sdk/README.md

```diff
@@ -70,7 +71,48 @@ to the `MeterProvider` as shown in the example [here](./Program.cs).
 
 Not supported.
 
-## Exemplar
+## ExemplarFilter
+
+OpenTelemetry .NET SDK has provided the following built-in `ExemplarFilter`s:
+
+* [AlwaysOnExemplarFilter](../../../src/OpenTelemetry/Metrics/Exemplar/AlwaysOnExemplarFilter.cs)
+* [AlwaysOffExemplarFilter](../../../src/OpenTelemetry/Metrics/Exemplar/AlwaysOffExemplarFilter.cs)
+* [TraceBasedExemplarFilter](../../../src/OpenTelemetry/Metrics/Exemplar/TraceBasedExemplarFilter.cs)
+
+Custom exemplar filters can be implemented to achieve filtering based on other criterion:
+
+* `ExemplarFilter` should derive from `OpenTelemetry.ExemplarFilter>` (which
+  belongs to the [OpenTelemetry](../../../src/OpenTelemetry/README.md) package)
+  and implement the `ShouldSample` method.
+
+One example is a filter, which filters all measurements of value lower
+than given threshold is given below. Such a filter prevents any measurements
+below the given threshold from ever becoming a `Exemplar`. Such filters could
+also incorporate the `TraceBasedExemplarFilter` condition as well, as storing
+exemplars for non-sampled traces may be undesired.
+
+```csharp
+public sealed class HighValueFilter : ExemplarFilter
```

Comment:
could use a better name suggestion. :)

This is filtering out measurements of "low numerical value"...